www/jxcore/app.js

```diff
@@ -42,7 +44,18 @@ app.use(function allowCrossDomain(req, res, next) {
 });
 
 app.get('/', function (req, res) {
-  res.render('ejs/index',  { user: 'user' + Math.floor(Math.random() * 100) });
+  var cryptomanager = require('./thalicryptomanager');
+  cryptomanager.getPublicKeyHash(function (publicKeyHash) {
```

Comment:
It really breaks my heart to see the user having to call cryptomanager themselves. Shouldn't the hash just be a property on the replication manager?